e2eshark/onnx/models/README.md

```diff
@@ -0,0 +1,130 @@
+## Writing a model.py File for a New Onnx Model
+
+### General Steps
+
+Making a model.py for an unknown model.onnx file can be done in a few steps:
+
+0. Copy boilerplate from another model.onnx file. 
+1. Figure out what the inputs look like.
+2. Make some randomized model inputs.
+3. Start an inference session and run it.
+4. Use E2ESHARK_CHECK to record inputs and outputs and apply post-processing if desired.
+
+Let's go into more depth into each step. 
+
+### 0. Copy boilerplate from another model.onnx file
+
+In addition to whatever additional python packages you choose to use, the boilerplate should minimally be:
+
+```python
+import numpy, torch, sys
+import onnxruntime
+
+# import from e2eshark/tools to allow running in current dir, for run through
+# run.pl, commutils is symbolically linked to allow any rundir to work
+sys.path.insert(0, "../../../tools/stubs")
+from commonutils import E2ESHARK_CHECK_DEF
+
+# Create an instance of it for this test
+E2ESHARK_CHECK = dict(E2ESHARK_CHECK_DEF)
+```
+
+### 1. Figure out what the inputs look like
+
+Frankly, I find it easiest to simply run the following python script to dump the inputs:
+
+```python
+import onnxruntime
+
+session = onnxruntime.InferenceSession("model.onnx", None)
```

Comment:
Where can I find the file `model.onnx` For a new model?

Reply:
The llama model kumar mentioned would be in azure (it might be in secure storage).

Reply:
It might also not be there yet

Reply:
got it